tests/unit/template_cassdc_test.go

```diff
@@ -97,6 +98,14 @@ var _ = Describe("Verify CassandraDatacenter template", func() {
 			// Server version and mgmt-api image specified
 			Expect(cassdc.Spec.ServerVersion).ToNot(BeEmpty())
 			Expect(cassdc.Spec.ServerImage).ToNot(BeEmpty())
+
+			// JVM heap options -- default to settings as defined in cassdc.yaml
+			var config Config
+			Expect(json.Unmarshal(cassdc.Spec.Config, &config)).To(Succeed())
+			Expect(config.JvmOptions).ToNot(BeNil())
+			Expect(config.JvmOptions.InitialHeapSize).To(Equal("800M"))
```

Comment:
As mentioned above, let's have defaults defined in `values.yaml` and the young generation has to be smaller than the total heap size.

Reply:
👍